[
   {
      "body" : "I haven't heard any discussion about such confusion myself. But I agree that ERROR is too strong here because this is part of `AcceptToMemoryPool` standard behaviour and doesn't need user's attention.\r\nOne or two days `debug.log` contains the following here:\r\n```\r\nERROR: AcceptToMemoryPool: inputs already spent\r\nERROR: AcceptToMemoryPool: nonstandard transaction: dust\r\nERROR: AcceptToMemoryPool: free transaction rejected by rate limiter\r\n```\r\nNone of these messages is an error, but standard information message about tx not being accepted to mempool. Looking at other `AcceptToMemoryPool` log messages in `debug.log`, I'd drop \"category/level\" prefix (ERROR: here) completely and change the message to contain \"not accepted\".\r\n",
      "created_at" : "2015-02-17T08:25:39Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/5794#issuecomment-74632454",
      "id" : 74632454,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/5794",
      "updated_at" : "2015-02-17T08:25:39Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/74632454",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/6848764?v=3",
         "events_url" : "https://api.github.com/users/paveljanik/events{/privacy}",
         "followers_url" : "https://api.github.com/users/paveljanik/followers",
         "following_url" : "https://api.github.com/users/paveljanik/following{/other_user}",
         "gists_url" : "https://api.github.com/users/paveljanik/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/paveljanik",
         "id" : 6848764,
         "login" : "paveljanik",
         "organizations_url" : "https://api.github.com/users/paveljanik/orgs",
         "received_events_url" : "https://api.github.com/users/paveljanik/received_events",
         "repos_url" : "https://api.github.com/users/paveljanik/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/paveljanik/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/paveljanik/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/paveljanik"
      }
   },
   {
      "body" : "Agreed, would be better if these are not ERROR.  I do remember discussions about this before.",
      "created_at" : "2015-02-25T08:44:23Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/5794#issuecomment-75924523",
      "id" : 75924523,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/5794",
      "updated_at" : "2015-02-25T08:44:23Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/75924523",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/126646?v=3",
         "events_url" : "https://api.github.com/users/laanwj/events{/privacy}",
         "followers_url" : "https://api.github.com/users/laanwj/followers",
         "following_url" : "https://api.github.com/users/laanwj/following{/other_user}",
         "gists_url" : "https://api.github.com/users/laanwj/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/laanwj",
         "id" : 126646,
         "login" : "laanwj",
         "organizations_url" : "https://api.github.com/users/laanwj/orgs",
         "received_events_url" : "https://api.github.com/users/laanwj/received_events",
         "repos_url" : "https://api.github.com/users/laanwj/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/laanwj/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/laanwj"
      }
   },
   {
      "body" : "I think we should get rid of all validation failure messages, and only\nreport thing when the final result of validation (in accepttomempool or\nprocessblock) is available, and report there based on debug settings.\n",
      "created_at" : "2015-02-25T10:16:52Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/5794#issuecomment-75936332",
      "id" : 75936332,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/5794",
      "updated_at" : "2015-02-25T10:16:52Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/75936332",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/548488?v=3",
         "events_url" : "https://api.github.com/users/sipa/events{/privacy}",
         "followers_url" : "https://api.github.com/users/sipa/followers",
         "following_url" : "https://api.github.com/users/sipa/following{/other_user}",
         "gists_url" : "https://api.github.com/users/sipa/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/sipa",
         "id" : 548488,
         "login" : "sipa",
         "organizations_url" : "https://api.github.com/users/sipa/orgs",
         "received_events_url" : "https://api.github.com/users/sipa/received_events",
         "repos_url" : "https://api.github.com/users/sipa/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/sipa/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/sipa/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/sipa"
      }
   },
   {
      "body" : "+1 from me. Just set up my first node (headless, no wallet) to contribute to the bitcoin movement and saw this in my logs. As someone said, this kind of error is terrifying when I'm investing masses of bandwidth downloading the blockchain and I'm too naive/inexperienced to know whether my install is screwing itself and/or the bitcoin network.",
      "created_at" : "2015-03-31T12:04:57Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/5794#issuecomment-88061680",
      "id" : 88061680,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/5794",
      "updated_at" : "2015-03-31T12:04:57Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/88061680",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/373456?v=3",
         "events_url" : "https://api.github.com/users/redpola/events{/privacy}",
         "followers_url" : "https://api.github.com/users/redpola/followers",
         "following_url" : "https://api.github.com/users/redpola/following{/other_user}",
         "gists_url" : "https://api.github.com/users/redpola/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/redpola",
         "id" : 373456,
         "login" : "redpola",
         "organizations_url" : "https://api.github.com/users/redpola/orgs",
         "received_events_url" : "https://api.github.com/users/redpola/received_events",
         "repos_url" : "https://api.github.com/users/redpola/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/redpola/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/redpola/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/redpola"
      }
   }
]
